packages/teleport/src/Kubes/KubeList/KubeList.tsx

```diff
@@ -61,6 +64,16 @@ function KubeList(props: Props) {
 
   const data = sortAndFilter(searchValue);
 
+  function openDialogue(kubeName: string) {
+    setShowConnectDialog(true);
+    setKubeName(kubeName);
+  }
+
+  function closeDialogue() {
+    setShowConnectDialog(false);
+    setKubeName('');
+  }
+
```

Comment:
i think we can just use the state of `kubeName` to display the dialogue

Reply:
yes, that is better and more cleaner.